lib/cocoapods/installer/xcode/pods_project_generator/target_installer.rb

```diff
@@ -99,6 +99,11 @@ def custom_build_settings
               settings.merge!('OTHER_LDFLAGS' => '', 'OTHER_LIBTOOLFLAGS' => '')
             end
 
+            if target.build_settings.generate.attributes.key?('SDKROOT')
+              puts "For target: #{target.name}, SDKROOT being set to #{target.build_settings.generate.attributes['SDKROOT']}"
```

Comment:
I dont think we want to notify the user for this at all, furthermore we can use `UI` class to do this instead of `puts`